adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/AdeleStyle.java

```diff
@@ -0,0 +1,32 @@
+package com.microservicesteam.adele.ticketmaster;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.immutables.value.Value.Style.BuilderVisibility;
+import org.immutables.value.Value.Style.ImplementationVisibility;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+@Target({PACKAGE, TYPE})
+@Retention(CLASS)
+@JsonSerialize
+@JsonDeserialize
+@Value.Style(
+    get = {"*"},
+    init = "*",
```

Comment:
@bearcheese yes, definitely I agree. I would create a separate modul now, and a separate lib later for defaults like this.